CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ if(CUDA_COMPILE)
             src/gpu/gpu_ray.cu src/gpu/gpu_ray.hpp
             src/gpu/gpu_sphere.cu src/gpu/gpu_sphere.hpp
             src/gpu/gpu_vec3.cu src/gpu/gpu_vec3.hpp
-            src/gpu/gpu_util.cu src/gpu/gpu_util.hpp)
+            src/gpu/gpu_util.cu src/gpu/gpu_util.hpp src/constant_medium.cpp src/constant_medium.cpp)
```

Comment:
remove this line change, assuming it got added automatically when you created the file but it is not needed here

Reply:
Resolved in ca316dc